### PR TITLE
[ch-15122] Netv2: Add satellite page

### PR DESCRIPTION
**Pull request by f0ssel**
Please feel free to commit directly to this PR making edits, this is a rough draft and I appreciate any improvements.

Still WIP on some assets and loose ends

### CODE REVIEW

File: admin/satellites.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+CNAME coder.example.com coder-ap-southeast.example.com
+
+A coder-us-east.example.com      xxx.xxx.xxx.xxx
+A coder-us-west.example.com      xxx.xxx.xxx.xxx
+A coder-eu-west.example.com      xxx.xxx.xxx.xxx
+A coder-ap-southeast.example.com xxx.xxx.xxx.xxx
+```
+
+Then developers can connect to `https://coder.example.com` and be serviced by
+the correct replica deployment based off the developer's location.
```
Comment (deansheather) [inline admin/satellites.md:45]:
```suggestion
the correct deployment based off the developer's location.
```

File: admin/satellites.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+## Geographic DNS
+
+Because it's always beneficially for developers to connect to the deployment
+closest to them for the lowest latency, DNS features such as anycast, GeoDNS, or
+GeoIP can be used to collect the coder deployments under a single hostname for a
+better developer experience. An example set of records with these features
+enabled could be:
+
+```text
+CNAME coder.example.com coder-us-east.example.com
```
Comment (deansheather) [inline admin/satellites.md:33]:
```suggestion
CNAME coder.example.com coder-primary.example.com
```

File: admin/satellites.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+A coder-eu-west.example.com      xxx.xxx.xxx.xxx
+A coder-ap-southeast.example.com xxx.xxx.xxx.xxx
+```
+
+Then developers can connect to `https://coder.example.com` and be serviced by
+the correct replica deployment based off the developer's location.
+
+## Creating a Satellite
+
+Creating a new satellite includes deploying the coder helm chart with the
```
Comment (deansheather) [inline admin/satellites.md:49]:
```suggestion
Creating a new satellite includes deploying the Coder Helm chart with the
```

File: admin/satellites.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+coderd:
+  satellite:
+    enable: "true"
+    primaryURL: "https://coder.example.com"
+    accessURL: "https://eu-west.example.com"
+  devurlsHost: "*.eu-west.example.com"
+  tls:
+    hostSecretName: "coder-satellite-cert"
+    devurlsHostSecretName: "coder-satellite-cert"
+```
```
Comment (deansheather) [inline admin/satellites.md:103]: Need example command.

File: admin/satellites.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+## Registering the Satellite
+
+Once the coder satellite has been deployed, it needs to be registered to the
+primary Coder deployment. This process ensures we can trust the requests coming
+from the satellite deployment.
+
+Using the Coder CLI, create a new satellite deployment:
+
+```bash
+coder satellites create [NAME] [SATELLITE_ACCESS_URL]
```
Comment (deansheather) [inline admin/satellites.md:114]:
```suggestion
coder satellites create <NAME> <SATELLITE_ACCESS_URL>
```

File: admin/satellites.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+## Removing a Satellite
+
+Using the Coder CLI, remove the satellite deployment:
+
+```bash
+coder satellites rm [NAME]
+```
+
+After removing a registered satellite deployment from Coder, the helm deployment
```
Comment (deansheather) [inline admin/satellites.md:137]:
```suggestion
After removing a registered satellite deployment from Coder, the Helm deployment
```

File: admin/satellites.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+should also be removed.
```
Comment (deansheather) [inline admin/satellites.md:138]: Need example command

File: admin/satellites/index.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+by acting as local proxies to the workspaces, keeping traffic from the
+developer's machine to the workspaces within the same region instead of
+requiring the traffic to cross regions back to the primary Coder deployment.
+
+Satellites, on the other hand, act as a secure reverse proxy to both Coder
+workspaces and the primary Coder deployment. Traffic meant for workspaces, such
+as web and SSH connections, are sent directly to the workspaces, while all other
+traffic is routed to the primary Coder deployment.
+
+[insert illustration here]
```
Comment (khorne3) [inline admin/satellites/index.md:21]: do we have this completed?

Comment (f0ssel) [reply admin/satellites/index.md:21]: looking for one / working on one now

Comment (f0ssel) [reply admin/satellites/index.md]: Added one, we can def add some polish but I made something passable

**Review by khorne3 — APPROVED** [review]

**Pull request merged**